vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -1009,12 +1001,14 @@ private void initHeaderFooterRenderer() {
     }
 
     private void setDimension(String dimension, String value) {
-        getElement().executeJs("requestAnimationFrame(e => "
-                + OVERLAY_LOCATOR_JS + ".$.overlay.style[$0]=$1)", dimension,
-                value);
+        getElement().executeJs(OVERLAY_LOCATOR_JS + ".$.overlay.style[$0]=$1",
+                dimension, value);
```

Comment:
Reverted this not so clean workaround from https://github.com/vaadin/flow-components/pull/4519